ttt/console_ui/menu.py

```diff
@@ -1,28 +1,37 @@
 import re
 
 import ttt.constants as constants
-from ttt.game.game import Game
+from ttt.game.board import Board
 from ttt.game.game_config import GameConfig
 from ttt.messages import welcome_message, player_type_message, marker_message, player_choice_message, end_game_message, \
-    play_again_message, colour_message
+    play_again_message, colour_message, load_game_message, load_game_error_message
+from ttt.persister.persister import Persister
 
 
 class Menu:
-    def __init__(self, console, game_config=GameConfig):
+    def __init__(self, console, game_config=GameConfig, persister=Persister()):
         self._console = console
         self._game_config = game_config(console)
+        self._persister = persister
 
-    def configure_game(self, game=Game):
+    def start(self):
         self._console.clear_output()
         self._console.output_message(welcome_message())
+        selection = self._console.get_validated_input(constants.TWO_OPTION_MENU_REGEX, constants.MENU_ERROR)
+        self._console.clear_output()
 
-        player_1 = self._setup_player(constants.PLAYER_1_NAME, constants.PLAYER_1_MARKER, None)
-        player_2 = self._setup_player(constants.PLAYER_2_NAME,
-                                      self._game_config.select_default_marker(player_1.get_marker()),
-                                      player_1.get_marker())
+        return self._configure_game() if selection is '1' else self.load_game()
```

Comment:
Would it make sense to replace `1` with a constant? I think it would help with consistency here.

Reply:
👍